apm-agent-plugins/apm-logging-plugin/apm-logging-plugin-common/src/test/java/co/elastic/apm/agent/loginstr/LoggingInstrumentationTest.java

```diff
@@ -161,43 +173,43 @@ protected boolean markersSupported() {
     @Test
     public void testShadingIntoOriginalLogsDir() throws Exception {
         setEcsReformattingConfig(LogEcsReformatting.SHADE);
-        initializeShadeDir("");
+        initializeReformattingDir("");
         runSimpleScenario();
     }
 
     @Test
-    public void testLazyShadeFileCreation() throws Exception {
-        initializeShadeDir("delayed");
+    public void testLazyEcsFileCreation() throws Exception {
+        initializeReformattingDir("delayed");
         logger.trace(TRACE_MESSAGE);
         logger.debug(DEBUG_MESSAGE);
-        assertThat(Files.exists(Paths.get(getShadeLogFilePath()))).isFalse();
+        assertThat(Files.exists(Paths.get(getLogReformattingFilePath()))).isFalse();
```

Comment:
> we can rely on assertJ assertions for files.

❤️ TIL

> As a further improvement, making the getLogReformattingFilePath() method return a Path instead of a String might be relevant too.

Some usages require the path as a String, since it's only within the test I will skip this one